safe_transaction_service/history/models.py

```diff
@@ -338,7 +339,7 @@ def create_from_tx_dict(
             nonce=tx["nonce"],
             to=tx.get("to"),
             value=tx["value"],
-            type=int(tx.get("type", "0x0"), 0),
+            type=tx.get("type", 0),
```

Comment:
Type is int in the model of `ethereumTx`, tx["type"] is always int?

Reply:
Yes, now it's parsed by web3.py (before it was an `HexStr`)